infrastructure/data/README.md

```diff
@@ -2,27 +2,154 @@
+
+The only [consideration](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_RestoreFromSnapshot.html#USER_RestoreFromSnapshot.Considerations) that applies to us from the former document is the point about parameter groups, since we are currently using a [custom parameter group](https://github.com/theopensystemslab/planx-new/blob/main/infrastructure/data/index.ts#L15), which only varies from the `postgres16` default in that it sets `rds.force_ssl = 0`.
+
+Regardless of whether you are going to restore the db via Pulumi or in the AWS console, the first two steps are the same:
+
+- Decide on the [snapshot](https://eu-west-2.console.aws.amazon.com/rds?region=eu-west-2#snapshots-list:) that you will restore the database from - probably the most recent one available for the relevant database.
+- If possible, take a snapshot of the database that you are intending to replace before proceeding (even if it's in some undesirable state). To do this, go to the relevant [database](https://eu-west-2.console.aws.amazon.com/rds?region=eu-west-2#databases:), navigate to `Maintenance and backups`, scroll down to `Snapshots`, and click the `Take snapshot` button.
+
+<!-- TODO: Is it right to recommend the Pulumi method, as below ?? -->
+
+Then, we recommend proceeding with the db restoration via the [Pulumi-managed method described below](#1-via-pulumi). However, if for some reason you anticipate problems with that approach, you don't want the current db to be deleted as part of the process, or you need more fine grained control of the new db's settings, then [using the AWS console](#2-via-the-aws-console) may make more sense.
+
+## 1. Via Pulumi
+
+The Pulumi class which controls our RDS db instance(s) is called `aws.rds.Instance` (provisioned [here](https://github.com/theopensystemslab/planx-new/blob/main/infrastructure/data/index.ts#L27), in [`index.ts`](/infrastructure/data/index.ts)).
+
+All we have to do to instruct Pulumi to spin up a new db based on the snapshot of our choice, is add the parameter `snapshotIdentifier` ([docs](https://www.pulumi.com/registry/packages/aws/api-docs/rds/instance/#snapshotidentifier_nodejs)), with the value as the name (not the ARN!) of the snapshot we want to restore from. See an example in [this commit](https://github.com/theopensystemslab/planx-new/pull/4392/commits/6ba7b2c8ff0b982514a787c3462d89655b8c1a10).
+
+Note that when we apply this change, Pulumi will first spin up a new db, with a new auto-generated/random name, and then delete the old one which it was managing. As above, if this is a problem, skip to the AWS method below. Otherwise, continue with the following steps:
+
+1.  Commit the `snapshotIdentifier: "snapshot-name"` line, push it up, get the PR approved and merge it in to `main` (this will have to be skipped if you're responding alone, but is good practice).
```

Comment:
nit: ```Commit the `snapshotIdentifier: "snapshot-name"` line (including ` [skip pizza]` to minimise CI time), push it up...```

Easy to forget in a stressed moment, but makes a difference to have quicker CI (and pizzas don't help us with Pulumi anyways!)

Reply:
Great idea!